main.go

```diff
@@ -428,33 +423,24 @@ func runWatcher(fileChangeCommand string, dirs []string, killServer bool) error
 
 	}
 
-	w.SetMaxEvents(1)
-
 	// Only files that match the regular expression during file listings
 	// will be watched.  Currently we watch java, js, and go files.
 	// We may add an environment variable to add to this list
 
-	w.AddFilterHook(watcher.RegexFilterHook(r, false))
+	//handle the ignore dirs by using a negative filter hook
+
+	// Start the watching process - it'll check for changes every "n" ms.
 
 	go func() {
 		for {
 			select {
 			case event := <-w.Event:
-				ControllerDebug.log("File watch event detected for:  " + event.Path)
-				if unwatchDir(event.Path) {
-					ControllerDebug.log("The path ", event.Path, " is not to be watched. Or this is a directory event which will not be watched.")
-				} else {
-					ControllerDebug.log("Determining if file or directory matches REGEX for:  " + event.Name())
-					if r.MatchString(event.Name()) {
+				fmt.Println("File watch event detected for:  " + event.String())
```

Comment:
Should this be ControllerDebug?

Reply:
I fixed the fmt.Println just now.  I have tested against multiple stacks and with different scenarios:
regular file change, ignored directories, no ignored directories.  I plan to spend more time documenting and doing tests for the doc prior to the release.